dataplane/standalone/port.cc

```diff
@@ -27,37 +29,42 @@ sai_status_t Port::create(_In_ uint32_t attr_count,
                           _In_ const sai_attribute_t* attr_list) {
   std::vector<sai_attribute_t> attrs(attr_list, attr_list + attr_count);
   std::vector<int> lanes;
-  std::string name;
+  std::string name = "eth" + std::to_string(Port::nextIdx);
+  Port::nextIdx += 1;
   sai_port_type_t type;
   for (auto attr : attrs) {
     switch (attr.id) {
       case SAI_PORT_ATTR_TYPE:
         type = static_cast<sai_port_type_t>(attr.value.s32);
         break;
       case SAI_PORT_ATTR_HW_LANE_LIST:
-        lanes = std::vector<int>(
-            attr.value.u32list.list,
-            attr.value.u32list.list + attr.value.u32list.count);
+        // lanes = std::vector<int>(
+        //     attr.value.u32list.list,
+        //     attr.value.u32list.list + attr.value.u32list.count);
 
-        for (auto port : Port::laneMap) {
-          if (port.second == lanes) {
-            name = port.first;
-            break;
-          }
-        }
+        // for (auto port : Port::laneMap) {
+        //   if (port.second == lanes) {
+        //     name = port.first;
+        //     break;
+        //   }
+        // }
         break;
     }
   }
+  std::string filename("/sys/class/net/" + name);
+  struct stat buffer;
+  bool exists = stat(filename.c_str(), &buffer) == 0;
```

Comment:
Be more descriptive about what this file is and what "exists == true" means.

Reply:
added comment